components/chart/search/textbox.tsx

```diff
@@ -119,29 +132,27 @@ export const Textbox = ({
       {autoCompleteList && autoCompleteList.length && showAutoComplete ? (
         <AutoComplete>
           <ul>
-            {autoCompleteList
+            {autoCompleteListNames
               .filter(
                 (val) =>
-                  val
+                  val.name
                     .toUpperCase()
                     .indexOf(
                       value
-                        .split(' ')
-                        [value.split(' ').length - 1].toUpperCase()
+                        .split(" ")
+                      [value.split(" ").length - 1].toUpperCase()
                     ) >= 0
               )
               .map((val, index) => (
                 <li
                   key={`AutoCompleteItem${index}`}
                   onClick={() => {
                     setShowAutoComplete(false);
-                    const words = value.split(' ');
-                    words.pop();
-                    const result = words.join(' ') + ' ' + val;
-                    onChange && onChange(result.substring(1), undefined); // Substring for removing the begining space resulted from join
+                    onListItemClick(val.id)
+                    onChange && onChange(val.name, undefined); // Substring for removing the begining space resulted from join
                   }}
                 >
-                  <span>{val}</span>
+                  <span>{' > ' + " "+ val.name}</span>
```

Comment:
Let's show only the name of the node, I think no need for ">"

Reply:
done